build/ci/cloudbuild.test.yaml

```diff
@@ -14,6 +14,26 @@
 
 steps:
 # Build protobuf to go bindings
+- id: lint
+  name: golang
+  entrypoint: 'bash'
+  args:
+  - -c
+  - |
+    set -e
+    c=`tail -c 1 deploy/storage/bigtable.version`
+    if [ "$c" == "" ]; then
```

Comment:
would the following be a stronger check? i.e., no newline at all in the file vs. newline not the last character of file

```
c=`grep -c '\n' deploy/storage/bigtable.version`
if ["$c" != "0" ]; then
  ...
```

Reply:
That's good idea. I tried grep '\n' but somehow it always give 0, even for files with multi line. Switched to use `cat deploy/storage/bigtable.version | wc -l`

Reply:
LG.. Sorry about that, per https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_04_02.html the reason is `Since newline is also a separator for the list of patterns, there is no way to match newline characters in a text.`

Reply:
thanks, good to know!